spec/features/fill_in_whole_form_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'rails_helper'
+RSpec.feature "Fill in whole form", js: true do
+  let(:respondents_details_page) { ET3::Test::RespondentsDetailsPage.new }
+  let(:claimants_details_page) { ET3::Test::ClaimantsDetailsPage.new }
+  let(:earnings_and_benefits_page) { ET3::Test::EarningsAndBenefitsPage.new }
+  let(:response_page) { ET3::Test::ResponsePage.new }
+  let(:your_representative_page) { ET3::Test::YourRepresentativePage.new }
+  let(:your_representatives_details_page) { ET3::Test::YourRepresentativesDetailsPage.new }
+  let(:employers_contract_claim_page) { ET3::Test::EmployersContractClaimPage.new }
+
+  scenario "correctly will flow without error" do
+
+    respondents_details_page.load
+
```

Comment:
This is one huge scenario.  Maybe lets not worry about it now, but lets address it when refactoring once everything is working.  You can use page objects for asserting text rather than fetching the text back and validating it yourself, which gives you that timing problem I mentioned before.  its fine for now though :-)

Reply:
It definitely seems too large to me, most of these elements are within other specs but I needed this document to ensure an end-to-end flow. Can I use the other specs and cut this document down to just page calls?

Reply:
totally understand the need for it - and i would never say remove it because its large - just maybe group some method calls together into helper methods etc..  Dont worry about it now though, we will have a right good refactoring at another stage - we are trying to get the positive flow finished, not get 100% excellent code - that can come afterwards :-)